express-rest-api/README.md

```diff
@@ -0,0 +1,7 @@
+# Express REST API example
+
+This example demonstrate how you can create test code using Mocha.js
+[Express](https://expressjs.com) used to create a server and [supertest](https://github.com/visionmedia/supertest) used for HTTP assertions, then Mocha validates the result. 
```

Comment:
Markdown prints these 2 lines together so it does not read very well. 
Perhaps better as `This example demonstrate how you can create tests on an [Express](https://expressjs.com) application using Mocha.js. Express is used to create a server and supertest is used for HTTP assertions inside the Mocha test runner.`